js-syntax-series/function-declarations.md

```diff
@@ -23,11 +23,11 @@ We see that one thing that separates them is the variable declaration and the la
 ## FunctionDeclarations
 One way to define functions in JavaScript is in the top level or in a block, starting with the keyword `function`. It's defined as a part of the [grammar of JavaScript](https://www.ecma-international.org/ecma-262/8.0/#prod-StatementListItem). Grammar is what defines what words are legal in what position in languages.
 
-A function declaration is of type `FunctionDeclaration` (analogous to "part of speech" tagging, as noun, verb, etc, in natural language), which is by grammar allowed to be child of `StatementListItem` which is a part of `StatementList`, which is a child of `Block`, etc, etc. It's all technical and wordy. But you can think of it like, if the line starts with the keyword `function` inside a block `{}` or at the top of your program, you probably have a function declaration.
+A function declaration is of type `FunctionDeclaration`, which is by grammar allowed to be child of `StatementListItem`. It's all technical and wordy. But you can think of it like this; if the line starts with the keyword `function` inside a block `{}` or at the top of your program, you probably have a function declaration.
```

Comment:
I get where you're coming from. But this might be a case where I think the details help illustrate the "complex rules" in play.

Reply:
Yeah, though I think the "complex rules" is pointed out nicely with the two `FunctionDeclaration` and `StatementListItem`, I fell off after the parentheses :) ... or I would flip it around. Lead with the "All technical wordy" so reader is prepared for a piece of text that they are not supposed to completely understand :)